### ISSUE TITLE
Third party plugins break after upgrading to 0.66

### ISSUE THREAD

**Issue by oschwald** [houseabsolute/perl-code-tidyall#82]
For instance, when using `Code::TidyAll::Plugin::ESLint` with a `cmd` argument, we now get:

```
unknown option 'cmd' for plugin 'ESLint mock-mf-interactive-api' at /home/greg/perl5/perlbrew/perls/perl-5.26.0/lib/site_perl/5.26.0/Code/TidyAll/Plugin.pm line 148.
```

**Comment by autarch** [comment]
Eek! I need to add a test using `Test::DependentModules` to this distro.

**Comment by autarch** [comment]
This is fixed in 0.67.

**Comment by oschwald** [comment]
Thanks!
